ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/flow_tracking.schema.yml

```diff
@@ -19,6 +19,21 @@ keys:
               - str
             min: 1
             max: 4294967295
+          hardware_offload:
+            type: dict
+            keys:
+              ipv4:
```

Comment:
can we maybe add the `ipv6` option in the same PR as it is really not much to code (and will prevent a future PR :) )
```
eos2(config-flow-tracking-sampled)#show cli commands | grep hardware
[no|default] hardware offload ( ( ipv4 [ ipv6 ] ) | ( ipv6 [ ipv4 ] ) )
[no|default] hardware offload threshold minimum SAMPLES samples
```

tested on ceos and the rendered CLI is always `ipv4` first:


```
ceos2(config)#show run | se flow tracking
flow tracking sampled
   hardware offload threshold minimum 2 samples
ceos2(config)#
ceos2(config)#flow tracking sampled
ceos2(config-flow-tracking-sampled)#hardware offload ipv6 ipv4
ceos2(config-flow-tracking-sampled)#exit
ceos2(config)#show run | se flow tracking
flow tracking sampled
   hardware offload ipv4 ipv6
   hardware offload threshold minimum 2 samples
```

Reply:
what you have done looks good